.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -151,13 +149,14 @@ After you have the version increments, the updated checkpoints and the updated c
 - [ ] Test that the newly built Zebra starts correctly, by running `~/.cargo/bin/zebrad`
 - [ ] Wait until the [Docker binaries have been built on `main`](https://github.com/ZcashFoundation/zebra/actions/workflows/continous-integration-docker.yml), and the quick tests have passed.
       (You can ignore the full sync and `lightwalletd` tests, because they take about a day to run.)
-- [ ] Publish the release to GitHub by disabling 'pre-release', then clicking "Publish Release"
+- [ ] [Publish the release to GitHub](https://github.com/ZcashFoundation/zebra/releases) by disabling 'pre-release', then clicking "Publish Release"
 - [ ] Wait until [the Docker images have been published](https://github.com/ZcashFoundation/zebra/actions/workflows/release-binaries.yml)
+- [ ] Test the Docker image using `docker run zfnd/zebra:1.0.0-rc.<version>` <!-- TODO: replace with `docker run zfnd/zebra` when we release 1.0.0 -->
```

Comment:
Shouldn't we test the Docker image before publishing it to Docker Hub? We have the latest version available in GCP, so we may want to test it locally before pulling the plug clicking `Publish Release`

Reply:
The goal of this test is to do exactly what the user will do to run Zebra. So testing GCP isn't enough, because our workflow might have a Docker Hub publishing bug.

When we implement the tests in #5168, we'll automate most of this testing by running the GCP image.

Reply:
I'll remove the build test to make the release check list shorter.

It's not something we have tier 1 support for, and we already have an automated test that does almost the same thing on every commit.